File: server/src/main/java/javaproject/server/dtos/responses/CategoryResponse.java
package javaproject.server.dtos.responses;

import com.fasterxml.jackson.annotation.JsonInclude;
import lombok.Builder;
import lombok.Data;

@Builder
@JsonInclude(JsonInclude.Include.NON_EMPTY)
public class CategoryResponse {
    private String id;
    private String name;
    private String description;

    public CategoryResponse(String id, String name, String description) {
        this.id = id;
        this.name = name;
        this.description = description;
    }

    public String getId() {
        return id;
    }

    public void setId(String id) {
        this.id = id;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getDescription() {
        return description;
    }

    public void setDescription(String description) {
        this.description = description;
    }
}

File: mobile/app/src/main/java/com/javaproject/mobile/data/model/responses/CategoryResponse.kt
package com.javaproject.mobile.data.model.responses

import com.google.gson.annotations.SerializedName

data class CategoryResponse (
    @SerializedName("id")
    val id: String,
    @SerializedName("name")
    val name: String,
    @SerializedName("description")
    val description: String
)